api-gateway/src/main/java/com/petclinic/bffapigateway/presentationlayer/v2/VetController.java

```diff
@@ -112,5 +114,14 @@ public Mono<ResponseEntity<VetResponseDTO>> getVetByVetId(@PathVariable String v
                 .defaultIfEmpty(ResponseEntity.notFound().build());
 
     }
+    //specialty
+    @SecuredEndpoint(allowedRoles = {Roles.ADMIN})
```

Comment:
It’s the vets who are supposed to be able to add photos. You should also add the vet role.